Use the correct macro name (AC_AUTOCONF_VERSION) in the CF_ACVERSION_CHECK macro.
The original macro name (AC_ACVERSION) leads to this error:

| autoreconf: running: aclocal --system-acdir=/poky/buildnew/tmp/work/i586-poky-linux/diffstat/1.57-r0/build/aclocal-copy/ --automake-acdir=/poky/buildnew/tmp/sysroots/x86_64-linux/usr/share/aclocal-1.12 --force
| aclocal: warning: autoconf input should be named 'configure.ac', not 'configure.in'
| configure.in:9: error: m4_defn: undefined macro: AC_ACVERSION
| acinclude.m4:989: CF_PROG_CC is expanded from...
| configure.in:9: the top level
| autom4te: m4 failed with exit status: 1

Upstream-Status: Inappropriate [embedded specific]
Signed-off-by: Bogdan Marinescu <bogdan.a.marinescu@intel.com>

diff --git a/aclocal.m4 b/aclocal.m4
index 144a074..fce6eba 100644
--- a/aclocal.m4
+++ b/aclocal.m4
@@ -13,15 +13,15 @@ dnl ------------------
 dnl Conditionally generate script according to whether we're using a given autoconf.
 dnl
 dnl $1 = version to compare against
-dnl $2 = code to use if AC_ACVERSION is at least as high as $1.
-dnl $3 = code to use if AC_ACVERSION is older than $1.
+dnl $2 = code to use if AC_AUTOCONF_VERSION is at least as high as $1.
+dnl $3 = code to use if AC_AUTOCONF_VERSION is older than $1.
 define([CF_ACVERSION_CHECK],
 [
 ifdef([m4_version_compare],
-[m4_if(m4_version_compare(m4_defn([AC_ACVERSION]), [$1]), -1, [$3], [$2])],
+[m4_if(m4_version_compare(m4_defn([AC_AUTOCONF_VERSION]), [$1]), -1, [$3], [$2])],
 [CF_ACVERSION_COMPARE(
 AC_PREREQ_CANON(AC_PREREQ_SPLIT([$1])),
-AC_PREREQ_CANON(AC_PREREQ_SPLIT(AC_ACVERSION)), AC_ACVERSION, [$2], [$3])])])dnl
+AC_PREREQ_CANON(AC_PREREQ_SPLIT(AC_AUTOCONF_VERSION)), AC_AUTOCONF_VERSION, [$2], [$3])])])dnl
 dnl ---------------------------------------------------------------------------
 dnl CF_ACVERSION_COMPARE version: 3 updated: 2012/10/03 18:39:53
 dnl --------------------
